Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -1034,7 +1068,7 @@ def get_alert_by_id_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
         context = {
             f'{INTEGRATION_CONTEXT_NAME}.Alert(val.ID && val.ID === obj.ID)': context_entry
         }
-        human_readable = tableToMarkdown(title, context_entry, ['ID', 'Name', 'Description', 'State', 'Severity'])
+        human_readable = tableToMarkdown(title, context_entry)
```

Comment:
filter null values, and add filter argument to the command.

Reply:
Done, and added to events (and consequently searches) and alerts